File: lecture003/fibopattern.java
import java.util.Scanner;

public class fibopattern{
    public static Scanner scn=new Scanner(System.in);
    public static void main(String[] args){
        

        int n;
        int a=0;
        int b=1;
        int sum=0;
        System.out.print("Enter no of layers : ");
        n=scn.nextInt();
        //System.out.print(a+" ");
        for(int i=0;i<n;i++){
            for(int j=0;j<=i;j++){
                System.out.print(a+" ");
                sum=a+b;
                a=b;
                b=sum;
                
            }
            System.out.println();
        }
    }
}

File: lecture003/fibopattern.cpp
#include<iostream>

using namespace std;

int main(int args,char**argv)
{   int n;
    int a=0;
    int b=1;
    int sum=0;
    
    cout<<"enter no of layers : ";
    cin>>n;
    
    for(int i=1;i<=n;i++)
    {
        for(int j=1;j<=i;j++)
        {
            cout<<a<<" ";
            sum=a+b;
            a=b;
            b=sum;
        }
        cout<<endl;
    }


}

